### ISSUE TITLE
Scrollable plot area and navigator

### ISSUE THREAD

**Issue by TorsteinHonsi** [highcharts/highcharts#11670]
#### Expected behaviour
Navigator to be placed at the bottom

#### Actual behaviour
Placed inside the scrollable plot area

#### Live demo with steps to reproduce
https://jsfiddle.net/highcharts/qsomL4kz/1/

#### Product version
v7.1.2

**Comment by TorsteinHonsi** [comment]
Note that `scrollablePlotArea` is currently only supported for Highcharts and Highcharts Gantt, so this is labeled an Enhancement.

**Comment by AntonyJos** [comment]
i guess both of these tasks are done.

@pawelfus , Thanks for adding the fix for navigator events.

Does the elements

`       '.highcharts-navigator-mask',
       '.highcharts-navigator-mask-outside',
       '.highcharts-navigator-outline',
       '.highcharts-navigator-handle',`

needed to be explicitly added to _moveFixedElements_  or just adding '.highcharts-navigator' would do the trick?
 
@TorsteinHonsi , Kindly review and consider merging these drafted changes so that it could be tested out with the help of cdn links

**Comment by pawelfus** [comment]
Just `.highcharts-navigator` container should be enough.

You can test current work-in-progress: https://jsfiddle.net/BlackLabel/zo85us90/

Link to PR's last commit CDN:
```
<script src="https://github.highcharts.com/869ef8ee2a32246fe3188985d30a1e5d223c8203/highstock.js"></script>
```

**Comment by AntonyJos** [comment]
_Observation made_ 

Fiddle: https://jsfiddle.net/qk94zeoc/1/

_**1)**_ Not able to see the complete Axis Height
Y Axises Range from 70 to 100 but even though i scroll div.highcharts-scrolling down the bottom part of the highCharts plot background is not visible.

![missing](https://user-images.githubusercontent.com/42953694/63263361-fe19b200-c2a5-11e9-92a9-a3a56570d903.PNG)

The part of the Axis from 70 - 80 is missing..

_**2)**_`navigator.margin` seems to have no effect . It does not create a gap between the x-Axis labels and the navigator position

**_3)_** The events on navigator like mousemove seems to propagate outside the highcharts-navigator container

![eventPropagation](https://user-images.githubusercontent.com/42953694/63263486-610b4900-c2a6-11e9-9139-0fdedc26bc02.PNG)

Can this be restricted to navigator container ?

**Comment by pawelfus** [comment]
ad 3) That's correct, it works exactly the same as without scrollable area: https://www.highcharts.com/stock/demo/basic-line - no plans for changing this now.

**Comment by AntonyJos** [comment]
@TorsteinHonsi / @pawelfus 

When can this be pushed to master?
Would this be available in the next release..

**Comment by TorsteinHonsi** [comment]
Sorry for the delay, it looks like it is ready to merge now. Will merge it once the CI tests have passed, then it will be available in the next release.

**Comment by danielroach** [comment]
Hi @TorsteinHonsi our team is waiting on this fix as we have an open ticket to enable "sticky axis headers" ie the functionality provided by `Scrollable plot area` and we are waiting on this fix to push that out.

Would you please be able to advise on an estimate for the release date?

**Comment by pawelfus** [comment]
We don't have ETA for the next release. Personally I think we should be ready within a month.

**Comment by cwragg** [comment]
HI @pawelfus and @TorsteinHonsi do you have a date to get this fix released? It is impacting our customers in a major way and affecting our business as a result.

**Comment by pawelfus** [comment]
Hi @cwragg - this issue should be fixed in v8. If you have found a bug, please create new issue. Thanks!
